docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,154 @@
+# How to integrate Autoware with your vehicle
+
+This page demonstrates how to integrate Autoware with a real vehicle.
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- Onboard computer that satisfies the prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
+- The following devices attached
+  - Actuator
+  - LiDAR
+  - Optional: Inertial measurement unit
+  - Optional: Camera
+  - Optional: GNSS
+
+## 2. Create maps
+
+You need both a pointcloud and a vector map to take full advantage of Autoware.
+Since mapping algorithm such as SLAM (simultaneous localization and mapping) is not implemented in the current Autoware, you may need to use 3rd party tools for this step.
+
+### Create a pointcloud map
+
+Use 3rd party tools such as the LiDAR-based SLAM package to create a pointcloud map. Autoware supports the `.pcd` format for this map.
+
+### Create vector map
+
+Autoware supports lanelet2 format for a vector map. Use 3rd party tools or [Vector Map Builder](https://tools.tier4.jp/) to get the `.osm` file.
+
+## 3. Create your meta-repository
+
+A recommended way to integrate Autoware with your real vehicle is to create a meta-repository for the vehicle.
+Create a forked repository of [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) (we refer to this as meta-repository) and clone the repository.
+
+```bash
+git clone git@github.com:YOUR_NAME/autoware.YOURS.git
```

Comment:
If we are going to describe how to clone the repository, then we should also explain how to fork it. We could either provide commands for both tasks (fork and clone), or provide links to GitHub documentation.